numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -1549,6 +1549,14 @@ gentype_@name@(PyObject *self, PyObject *args, PyObject *kwds)
 }
 /**end repeat**/
 
+static PyObject *
+dunder_round(PyObject *self, PyObject *args, PyObject *kwds)
```

Comment:
What do `args` and `kwds` hold here?

Reply:
https://docs.python.org/3/library/functions.html#round

Probably `ndigits`. I should check that it's not present before passing it on.

Reply:
I changed this to make it consistent with Python, too. It now emits a `float` with the second argument provided, even if it's `0`.